src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -35,15 +35,46 @@ public static class Win32ApplicationExtensions
         }
     }
 
+    /// <summary>
+    /// Platform-specific options which apply to Windows.
+    /// </summary>
     public class Win32PlatformOptions
     {
+        /// <summary>
+        /// Deferred renderer would be used on Windows when set to true. Immediate renderer would be used when set to false. The default value is true.
+        /// </summary>
+        /// <remarks>
+        /// Avalonia has two different renderers - Immediate and Deferred.
+        /// Immediate re-renders the whole scene when some element is changed on the scene. Deferred re-renders only changed elements.
+        /// </remarks>
         public bool UseDeferredRendering { get; set; } = true;
-        
+
+        /// <summary>
+        /// Enables ANGLE for Windows. For every Windows which is above Win 7, the default is true,otherwise-false.
```

Comment:
```suggestion
        /// Enables ANGLE for Windows. For every Windows version that is above Windows 7, the default is true otherwise it's false.
```